t/t2262-job-list-purge.t

```diff
@@ -0,0 +1,181 @@
+#!/bin/sh
+
+test_description='Test flux job list purge inactive'
+
+. $(dirname $0)/sharness.sh
+
+export FLUX_CONF_DIR=$(pwd)
+test_under_flux 4 job
+
+RPC=${FLUX_BUILD_DIR}/t/request/rpc
+
+fj_wait_event() {
+  flux job wait-event --timeout=20 "$@"
+}
+
+get_completed_count() {
+    count=`flux jobs --stats-only 2>&1 | awk '{print \$3}'`
+    echo ${count}
+}
+
+get_failed_count() {
+    count=`flux jobs --stats-only 2>&1 | awk '{print \$5}'`
+    echo ${count}
+}
+
+# Return the expected jobids list in a given state:
+#   "inactive", "completed", "failed"
+#
+state_ids() {
+    for f in "$@"; do
+        cat ${f}.ids
+    done
+}
+
+# Return the expected count of jobs in a given state (See above for list)
+#
+state_count() {
+    state_ids "$@" | wc -l
+}
+
+test_expect_success 'job-list: submit 6 jobs' '
+        #  Create `hostname` and `sleep` jobspec
+        #
+        flux jobspec --format json srun -N1 hostname > hostname.json &&
+        flux jobspec --format json srun -N1 sleep 300 > sleeplong.json &&
```

Comment:
We probably should avoid using `flux jobspec` in any new tests, I think we were considering deprecating it?

Reply:
ahhh, this style exists in a number of other test files already, which is why I cut & pasted it over.  Perhaps we should clean up those other test files too as well.